d2bs/kolbot/default.dbj

```diff
@@ -22,6 +23,7 @@ include("common/Prototypes.js");
 include("common/Runewords.js");
 include("common/Storage.js");
 include("common/Town.js");
+include("common/Team.js");
```

Comment:
This Team.js should not be in common folder as it is not a common lib.

Reply:
Yes i agree. I'll make this more abstract, i have an general idea but it isn't worked out in my head yet. So i'll need some time before i do so